docs/quickstart-make.md

```diff
@@ -31,7 +31,7 @@ exit
 ## power on six machines from the data center instance
 
 ```sh
-bash virtual-datacenter/power-on.sh
+bash colony-vagrant/virtual-datacenter/power-on.sh
```

Comment:
It seems the folder didn't move inside though. This was relative to the project's root since that's the requirement: clone the repo.

Reply:
colony-vagrant is related to the project root; virtual-datacenter is not in the root!

Reply:
Right, so the root folder of `github.com/konstructio/colony-vagrant` is the root. If you `ls` in that folder, you'll see `Vagrantfile` in it:

```
$ ls
Makefile                helpers.rb              switches.rb
README.md               laptop                  virtual-datacenter
Vagrantfile             mikrotik
docs                    servers.rb
```

If I then from the root try to run the script, the path is wrong:

```bash
$ bash colony-vagrant/virtual-datacenter/power-on.sh
bash: colony-vagrant/virtual-datacenter/power-on.sh: No such file or directory
```